lib/types-database.ts

```diff
@@ -0,0 +1,9 @@
+export interface formUser {
```

Comment:
We have a types folder. Would probably put that there instead of a separate file

Reply:
Sorry I meant the types.tsx. It should probably be in there

Reply:
Removed the `getForm` function, which removed the need for this interface.